tests/test_jx/test_metadata.py

```diff
@@ -43,14 +43,14 @@ def test_meta(self):
         }
         self.utils.send_queries(pre_test)
 
-        test = set_default(test, {
+        test = {
             "query": {
                 "select": ["name", "table", "type", "nested_path"],
-                "from": "meta.columns",
-                "where": {"eq": {"table": table_name}}
```

Comment:
... you can assume the `where` clause will only filter on the table name, or the column name.  That may make the SQL generation easier.